test/modules/TestResultCoordinator/service/TestResultEventReceivingService.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Service
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Common;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.EventHubs;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    class TestResultEventReceivingService : BackgroundService
+    {
+        readonly ILogger logger = ModuleUtil.CreateLogger(nameof(TestResultEventReceivingService));
```

Comment:
very nit: all our other loggers are capitalized - we should stay consistent

Reply:
it is small case here due to it is class variable, not static.